src/components/prescriptions/PrescriptionAddComponent.vue

```diff
@@ -176,19 +184,6 @@
                         @include text-button($font-weight-semiBold);
                     }
                 }
-
```

Comment:
why deleted

Reply:
if it not used in form i think i can delete it.